packages/xflow-core/src/xflow-main/components/canvas.tsx

```diff
@@ -69,6 +69,15 @@ export const XFlowCanvas: React.FC<IProps> = props => {
   const rootClz = useXflowPrefixCls('canvas-root')
   const canvasClz = useXflowPrefixCls('x6-canvas')
 
+  // 监听rootRef的宽度变化，更新画布的宽度
```

Comment:
perhaps to add these code to graph hooks 
see below:
https://github.com/antvis/XFlow/blob/24db2b17933f17999b5cf811d96d063b180cb066/packages/xflow-core/src/hooks/contributions/graph.ts